### PR TITLE
Initial commit of ImageCapture code

### PR DESCRIPTION
**Pull request by karinlackrif**

### CODE REVIEW

**Comment by karinlackrif** [comment]
Hey @khawkins  and @maliroteh-sf, can you please review this PR?
For some reason I can't add reviewers.

File: ImageCapture/scripts/apex/hello.apex
Comment (maliroteh-sf) [inline ImageCapture/scripts/apex/hello.apex:1]: You don't need this file... in fact you can delete the entire `scripts` folder

File: ImageCapture/scripts/soql/account.soql
Comment (maliroteh-sf) [inline ImageCapture/scripts/soql/account.soql:1]: same here... you don't need this file.

**Review by maliroteh-sf — COMMENTED** [review]
Skimmed through the changes and left some comments. Overall looks good, but it be good to take care of the TODOs before merging the PR. I also would like to wait for @khawkins  to take a look too.

File: ImageCapture/force-app/main/default/lwc/imagePainter/imagePainter.js
Comment (karinlackrif) [reply ImageCapture/force-app/main/default/lwc/imagePainter/imagePainter.js:59]: No, only in the next release, thanks for catching this.

File: ImageCapture/force-app/main/default/lwc/imagePainter/imagePainter.js
Comment (karinlackrif) [reply ImageCapture/force-app/main/default/lwc/imagePainter/imagePainter.js:196]: Yes you are right, thanks for catching this.

**Comment by karinlackrif** [comment]
Thanks @maliroteh-sf!
I will wait for @khawkins to approve.

File: UploadAndAnnotateImages/force-app/main/default/lwc/imageAnnotate/imageAnnotate.js
Comment (khawkins) [inline UploadAndAnnotateImages/force-app/main/default/lwc/imageAnnotate/imageAnnotate.js:6]: Nit: this line generates the TS1219 warning in VSCode, regarding experimental decorators. We should add a `jsconfig.json` [configuration](https://code.visualstudio.com/docs/languages/jsconfig) under `lwc/` that sets:

```json
"compilerOptions": {
    "experimentalDecorators": true
}
```

So this warning won't show up in VSCode for any of the LWCs.

File: UploadAndAnnotateImages/force-app/main/default/lwc/imageCapture/imageCapture.js
Comment (khawkins) [inline UploadAndAnnotateImages/force-app/main/default/lwc/imageCapture/imageCapture.js:54]: Nit: you could simplify this by taking out the ternary in favor of `return this.toastType != null;`

**Review by khawkins — CHANGES_REQUESTED** [review]
This looks fine over all. The things I'd like to see updated, prior to merge, are:

- Add the `jsconfig.json` configuration for `experimentalDecorators`, per [this comment](https://github.com/forcedotcom/LWC-Mobile-Samples/pull/10#discussion_r1069882021).
- Rebase against the HEAD of `main`, and run `yarn install && yarn format:fix` at the top of the repo.
- Commit those JS format updates as part of this PR.

File: UploadAndAnnotateImages/force-app/main/default/lwc/imageAnnotate/imageAnnotate.js
Comment (karinlackrif) [reply UploadAndAnnotateImages/force-app/main/default/lwc/imageAnnotate/imageAnnotate.js]: Thanks @khawkins.
This file exists locally, but it's ignored according to `.gitignore` (and `.forceignore`).
Should I remove it from the ignore files and add it to the repo?

File: UploadAndAnnotateImages/force-app/main/default/lwc/imageCapture/imageCapture.js
Comment (karinlackrif) [reply UploadAndAnnotateImages/force-app/main/default/lwc/imageCapture/imageCapture.js]: Right, thanks!

File: UploadAndAnnotateImages/force-app/main/default/lwc/imageAnnotate/imageAnnotate.js
Comment (khawkins) [reply UploadAndAnnotateImages/force-app/main/default/lwc/imageAnnotate/imageAnnotate.js]: Yes please, let's make it a part of the repo. Let's remove the entry from `.gitignore`, though I think it's okay to remain in `.forceignore`, as it's not necessary for that file to end up in your Org source. Just the repo itself (for development).

Comment (karinlackrif) [reply UploadAndAnnotateImages/force-app/main/default/lwc/imageAnnotate/imageAnnotate.js]: Done:)

File: UploadAndAnnotateImages/package.json
Comment (khawkins) [inline UploadAndAnnotateImages/package.json:15]: This doesn't work with `npm install`, presumably because Husky is meant to install hooks into the `.git/` repo, but this package is not associated with a repository at its root.

```bash
> salesforce-app@1.0.0 postinstall
> husky install

.git can't be found (see https://git.io/Jc3F9)
npm ERR! code 1
npm ERR! path /<My Repo Dir>/LWC-Mobile-Samples/UploadAndAnnotateImages
npm ERR! command failed
npm ERR! command sh -c -- husky install

npm ERR! A complete log of this run can be found in:
npm ERR!     /<My Home Dir>/.npm/_logs/2023-01-25T20_20_19_284Z-debug-0.log
```

We should look at either look at updating the husky configuration to support the subfolder, similar to [this example](https://scottsauber.com/2021/06/01/using-husky-git-hooks-and-lint-staged-with-nested-folders/), or remove the Husky requirement altogether.

**Review by khawkins — APPROVED** [review]

**Review by khawkins — CHANGES_REQUESTED** [review]
The only remaining changes for me is to rework the Husky configuration in UploadAndAnnotateImages. Otherwise, this all looks fine to me.

File: UploadAndAnnotateImages/package.json
Comment (karinlackrif) [reply UploadAndAnnotateImages/package.json:15]: Thanks! I followed the example you shared and it seems to work now.

**Review by khawkins — APPROVED** [review]

**Pull request merged**
